src/swig_python/python_extra.py_in

```diff
@@ -102,9 +102,13 @@ def _script_push_from_bytes_len_fn(data, flags):
             break
     return push_len + opcode_len
 script_push_from_bytes = _wrap_bin(script_push_from_bytes, _script_push_from_bytes_len_fn, resize=True)
-def _spkcsv2of2_1_len_fn(sigs, csv_blocks, flags):
+def _spkcsv2of2_1_opt_len_fn(sigs, csv_blocks, flags):
```

Comment:
nit: s/sigs/pubkeys/ (also below)

however here it does not really matter so to me it's fine to leave it as it is

Reply:
This was wrong in a couple of places so I updated them all, thanks!